src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -19,12 +19,21 @@
  */
 
 public final class StringHelper {
+    public static final String EMPTY = "";
```

Comment:
I don't think this is worth being extracted out. `""` is clear enough to be understood as an empty string.